packages/expo-sqlite/ios/ExpoSQLite.podspec

```diff
@@ -13,13 +13,16 @@ Pod::Spec.new do |s|
   s.platform       = :ios, '13.0'
   s.source         = { git: 'https://github.com/expo/expo.git' }
   s.static_framework = true
-
+  
   s.dependency 'ExpoModulesCore'
+  s.dependency 'sqlite3'
```

Comment:
nit: comment why we need the custom sqlite3 rather than the ios builtin one. (as you mentioned to me before, because `sqlite3_auto_extension` is not supported from the ios builtin sqlite)